service/matching/matchingEngine.go

```diff
@@ -615,13 +615,12 @@ func (e *matchingEngineImpl) listTaskListPartitions(
 
 	var partitionHostInfo []*workflow.TaskListPartitionMetadata
 	for _, partition := range partitions {
-		if host, err := e.getHostInfo(partition); err != nil {
-			partitionHostInfo = append(partitionHostInfo,
-				&workflow.TaskListPartitionMetadata{
-					Key:           common.StringPtr(partition),
-					OwnerHostName: common.StringPtr(host),
-				})
-		}
+		host, _ := e.getHostInfo(partition)
```

Comment:
Do we no longer need the the err if?

Reply:
If there is no error we return empty string for host info. So, we still can get partition name even if host information for that partition name cannot be found.